packages/frontend-api/src/Resources/config/graphql-types/ProductFilter/ProductFilterOptionsDecorator.types.yaml

```diff
@@ -0,0 +1,24 @@
+ProductFilterOptionsDecorator:
+    type: interface
+    decorator: true
+    config:
+        description: "Represents a product filter options"
+        fields:
+            minimalPrice:
+                type: "Money!"
+                description: "Minimal price of products for filtering"
+            maximalPrice:
+                type: "Money!"
+                description: "Maximal price of products for filtering"
+            brands:
+                type: "[BrandFilterOption!]"
+                description: "Brands filter options"
+            flags:
+                type: "[FlagFilterOption!]"
+                description: "Flags filter options"
+            inStock:
+                type: "Int!"
+                description: "Number of products in stock that will be added to products result"
```

Comment:
Isn't it more like `Number of products in stock`? 
It's always an absolute number of products in stock based on the provided criteria (eg. it always narrows the list, never expand)

Reply:
9c7db51a92c2239b047dad7635fbe1678a1010cc